BraveShared/Analytics/DAU.swift

```diff
@@ -56,6 +63,7 @@ public class DAU {
     }
     
     @objc public func sendPingToServerInternal() {
+        processingPing = true
         guard let paramsAndPrefs = paramsAndPrefsSetup() else {
```

Comment:
Check for `processingPing ` also and set `processingPing` as true after the guard.

Reply:
Added the former, but why the latter?

Reply:
It will always be processingPing ie true if paramsAndPrefsSetup failed for today but not after midnight